clayui.com/static/js/code-collapse.js

```diff
@@ -0,0 +1,35 @@
+if (document.body) {
```

Comment:
The formatting seems a little off in this file. Can you run prettier again on it?